idea_town/frontend/static-src/app/templates/contributor-view.js

```diff
@@ -0,0 +1,9 @@
+export default `
```

Comment:
This might sound backward, but: For simple templates like this - i.e. under 20 lines, no mustache logic - you might just skip the separate .js and just embed the template string directly in the view. [That felt pretty convenient (and React-like) to me in my c25k project](https://github.com/lmorchard/c25k-web/blob/master/src/js/views/Workout.js#L9), keeping the small markup fragments close to the bindings & events declarations.